test_utils/src/lib.rs

```diff
@@ -936,6 +938,274 @@ pub fn mock_signed_certificate(
             .unwrap();
         votes.push((pk.clone(), sig))
     }
-    let cert = Certificate::new(&committee(None), header, votes).unwrap();
+    let cert = Certificate::new(committee, header, votes).unwrap();
     (cert.digest(), cert)
 }
+
+use rand::rngs::OsRng;
```

Comment:
should this go at the top?

Reply:
They don't have to go at the top. Depending on the situation it may may sense to have a use inside a function or a block. In this case it probably makes the most sense at the top of the file :)